src/utils/utils.py

```diff
@@ -132,6 +132,9 @@ def write_solution_to_file(
         for i in model_data.resources:
             job_sol[j].extend(list(solution[j, i]))
 
+    dir_path = "/".join(solution_file_path.split("/")[:-1])
+    Path(dir_path).mkdir(parents=True, exist_ok=True)
```

Comment:
I believe this is what you want.
```suggestion
    Path(solution_file_path).parent.mkdir(parents=True, exist_ok=True)
```